.github/workflows/release.yml

```
@@ -394,10 +394,218 @@
+          SBOM files are included in CycloneDX format for dependency tracking and compliance.
+          
+          ## Security Scanning
+          
+          All binaries can be scanned with standard enterprise security tools.
+          EOF
+          
+          # Create PowerShell installation script
+          cat > install.ps1 << 'EOF'
+          # Scopes Offline Installation Script for Windows PowerShell
+          # This script installs Scopes from a pre-downloaded offline package
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)